### PR TITLE
Week4

### PR DESCRIPTION
**Pull request by JenMcD-star**
week 4 assignment

### CODE REVIEW

File: 02-express-tutorial/09-middleware-use.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+const express = require('express')
+const app = express()
+const logger = require('./logger')
+const authorize = require('./authorize')
+//  req => middleware => res
+app.use([authorize, logger])
```
Comment (taylor-knapp) [inline 02-express-tutorial/09-middleware-use.js]: fwiw, just make sure you understand that the order of middleware is important - the example logs each request before it authorizes, while this order authorizes first so nothing is ever logged if the request is unauthorized.

File: 02-express-tutorial/15-router-controller.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
+let { people } = require('../data')
```
Comment (taylor-knapp) [inline 02-express-tutorial/15-router-controller.js]: The file reference here doesn't exist - it should be `require('./data')`.

**Review by taylor-knapp — APPROVED** [review]
Great work! 👏 🎉  Two minor comments.

File: 02-express-tutorial/15-router-controller.js (hunk 2 above)
Comment (JenMcD-star) [reply 02-express-tutorial/15-router-controller.js]: Ah, I added that in the final folder to see what the instructor version looked like compared to mine and I guess I just copied it into mine as well!

**Pull request merged**